python/pyspark/pandas/indexing.py

```diff
@@ -1246,7 +1255,8 @@ def _select_cols_by_iterable(
                     % (len(cast(Sized, cols_sel)), len(self._internal.column_labels))
                 )
             if isinstance(cols_sel, pd.Series):
-                if not cols_sel.index.sort_values().equals(self._psdf.columns.sort_values()):
+                if not cols_sel.index.sort_values().equals(
+                        self._psdf.columns.sort_values()):  # type: ignore
```

Comment:
Shall we adjust the indent of this line?

Reply:
Updated